setup.py

```diff
@@ -23,7 +23,7 @@
         "atlas-commons>=0.1.4",
         "click>=7.0",
         "numpy>=1.15.0",
-        "scipy>=1.4.1",
```

Comment:
since this is just a lower bound, and not a pin, I'm that the newest scipy wasn't being picked up - correct fix, regardless.